mappings/net/minecraft/network/packet/s2c/query/MetadataS2CPacket.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_aleeatzq net/minecraft/network/packet/s2c/query/QueryResponseS2CPacket
+CLASS net/minecraft/unmapped/C_aleeatzq net/minecraft/network/packet/s2c/query/MetadataS2CPacket
```

Comment:
`QueryMetadata`?

Reply:
this seems like a needless breach of convention to me